homeassistant/components/alexa/handlers.py

```diff
@@ -924,3 +943,53 @@ def temperature_from_object(hass, temp_obj, interval=False):
     )
 
     return response
+
+
+@HANDLERS.register(('Alexa.ChannelController', 'ChangeChannel'))
+async def async_api_changechannel(hass, config, directive, context):
+    """Process a change channel request."""
+    channel = 0
+    entity = directive.entity
+
+    if 'number' in directive.payload['channel'].keys():
+        channel = directive.payload['channel']['number']
+    elif 'callSign' in directive.payload['channel'].keys():
+        channel = directive.payload['channel']['callSign']
+    elif 'name' in directive.payload['channelMetadata'].keys():
+        channel = directive.payload['channelMetadata']['name']
+
+    data = {
+        ATTR_ENTITY_ID: entity.entity_id,
+        media_player.const.ATTR_MEDIA_CONTENT_ID: channel,
+        media_player.const.ATTR_MEDIA_CONTENT_TYPE: 'channel'
+    }
+
+    await hass.services.async_call(
+        entity.domain, media_player.const.SERVICE_PLAY_MEDIA,
+        data, blocking=False, context=context)
+
+    return directive.response()
+
+
+@HANDLERS.register(('Alexa.ChannelController', 'SkipChannels'))
+async def async_api_skipchannel(hass, config, directive, context):
+    """Process a change channel request."""
+
+    channel = int(directive.payload['channelCount'])
+    entity = directive.entity
+
+    data = {
+        ATTR_ENTITY_ID: entity.entity_id
+    }
+
+    if channel > 0:
+        await hass.services.async_call(
+            entity.domain, SERVICE_MEDIA_NEXT_TRACK,
+            data, blocking=False, context=context)
+
+    if channel < 0:
+        await hass.services.async_call(
+            entity.domain, SERVICE_MEDIA_PREVIOUS_TRACK,
+            data, blocking=False, context=context)
+
+    return directive.response()
```

Comment:
```suggestion
    response = directive.response()
    response.add_context_property(
        {
            "namespace": "Alexa.ChannelController",
            "name": "channel",
            "value": {
              "number": "1234",
              "callSign": "callsign1",
              "affiliateCallSign": "callsign2"
            }
        )
        
        return response
```
According the the Alexa API:

> You must send an Response event if a request to change or skip channels was successful. The response should include a context object that reports the channel after the request completes.